src/Components/QuestionsDisplay.js

```diff
@@ -1,122 +1,153 @@
+      wordType = "noun";
+    } else if (wordTypeSymbol === "v") {
+      wordType = "verb";
+    } else if (wordTypeSymbol === "adj") {
+      wordType = "adjective";
+    } else if (wordTypeSymbol === "adv") {
+      wordType = "adverb";
+    } else {
+      return null;
     }
+  };
 
-    return (
-        <div className="App">
-            <h2>Question {questionNumber}:</h2>
-            {data.map((item, index) => {
-                homophone = item.word;
-                return (
-                    <div key={index}>
-                        {evaluateWordType(item)}
-                        <p><span className="wordType">{wordType} - </span>{definition}</p>
-                        <form action="#" className="questionForm">
-                            <fieldset onChange={(event) => handleUserAnswerSelection(event)}>
-                                <legend className="sr-only">Click on the word that matches the definition</legend>
-                                <label htmlFor="wordOne">{randomQuestionPosition === 1 ? randomWord : item.word}</label>
-                                <input type="radio" id="wordOne" name="homophone" value={randomQuestionPosition === 1 ? randomWord : item.word} checked={wordOneChecked} onChange={(event) => handleChange(event)} />
-                                <label htmlFor="wordTwo">{randomQuestionPosition === 1 ? item.word : randomWord}</label>
-                                <input type="radio" id="wordTwo" name="homophone" value={randomQuestionPosition === 1 ? item.word : randomWord} checked={wordTwoChecked} onChange={(event) => handleChange(event)} />
-                            </fieldset>
-                        </form>
-                        <button onClick={handleNextQuestion} className="nextQ">
-                            <FontAwesomeIcon
-                                className="buttonIcon"
-                                icon={faChevronRight}
-                                aria-hidden="true"
-                            />
-                            <p>Next</p>
-                        </button>
-                    </div>
-                );
-            })}
-        </div>
-    )
-}
+  return (
+    <div className="wrapper">
+      <h2 className="questionNumber">Question {questionNumber}:</h2>
+      {data.map((item, index) => {
+        homophone = item.word;
+        return (
+          <div key={index}>
+            {evaluateWordType(item)}
+            <p>
+              <span className="wordType">{wordType} </span>
+              {definition}.
+            </p>
+            <form action="#" className="questionForm">
+              <fieldset onChange={(event) => handleUserAnswerSelection(event)}>
+                <legend className="sr-only">
+                  Click on the word that matches the definition
+                </legend>
+                <div className="wordOneContainer">
+                  <p>Lorem ipsum dolor sit amet consectetur</p>
```

Comment:
We can move the text in the p tag (lorem ipsum) inside the label so that the entire page is clickable